tutorials/tutorial-hello-timescale.md

```diff
@@ -0,0 +1,798 @@
+ 2016-01-28 00:00:00 | 383326
+ 2016-01-29 00:00:00 | 414039
+ 2016-01-30 00:00:00 | 435369
+ 2016-01-31 00:00:00 | 361505
+ 2017-11-17 00:00:00 |      2
+(32 rows)
+```
+
+#### What is the average fare amount for 2+ passengers?
+
+But this initial analysis is incomplete. We also need to understand how long each
+ride is. After all, if we're looking to lessen the impact on the environment, we
+probably want to discourage idle taxis and short trips.
+
+One way we can glean insight into the duration of trips is by looking into 
+the daily average fare amount for rides with 2+ passengers. Once 
+again, this is a simple four line SQL query with some conditional statements,
+shown in the query below:
+
+```sql
+-- What is the daily average fare amount for rides with 2 or more passengers
+-- for first 7 days?
+SELECT date_trunc('day', pickup_datetime) as day, avg(fare_amount) FROM rides WHERE passenger_count > 1 AND pickup_datetime < '2016-01-08' GROUP BY day ORDER BY day;
+```
+
+>:TIP: Queries like the ones above execute up to 20x faster on TimescaleDB than on a vanilla PostgreSQL database, thanks to Timescale’s automatic time and space partitioning.
+
+Your result should look like this:
+
+```sql
+         day         |         avg         
+---------------------+---------------------
+ 2016-01-01 00:00:00 | 13.3990821679715529
+ 2016-01-02 00:00:00 | 13.0224687415181399
+ 2016-01-03 00:00:00 | 13.5382068607068607
+ 2016-01-04 00:00:00 | 12.9618895561740149
+ 2016-01-05 00:00:00 | 12.6614611935518309
+ 2016-01-06 00:00:00 | 12.5775245695086098
+ 2016-01-07 00:00:00 | 12.5868802584437019
+(7 rows)
+```
+
+#### How many rides took place by each rate type?
+
+Of course, the fare amount only gives us a certain amount of insight. Some fares
+are pre-set, regardless of the length. For example, trips to and from the airport are 
+a flat rate from within the city. So, let's examine the breakdown of 
+rides by ride type. This is also a fairly straightforward SQL query:
+
+```sql
+-- How many rides of each rate type took place in the month?
+SELECT rate_code, COUNT(vendor_id) as num_trips FROM rides
+WHERE pickup_datetime < '2016-02-01'
+GROUP BY rate_code ORDER BY rate_code;
+```
+
+After running the query above, you’ll get the following output, 
+which shows how many rides of each rate code took place:
+
+```sql
+ rate_code | num_trips 
+-----------+-----------
+         1 |  10626315
+         2 |    225019
+         3 |     16822
+         4 |      4696
+         5 |     33688
+         6 |       102
+        99 |       216
+(7 rows)
+```
+
+While that’s technically correct, you’d like to use something more human 
+readable. To do that, we can use the power of SQL joins, and combine
+these results with the contents of the `rates` table, like in the query below:
+
+```sql
+-- How many rides of each rate type took place?
+-- Join rides with rates to get more information on rate_code
+SELECT rates.description, COUNT(vendor_id) as num_trips FROM rides
```

Comment:
Make this query fully vertical: `FROM`, `ORDER BY` conditions should be on separate lines